components/mosart/src/riverroute/RunoffMod.F90

```diff
@@ -77,6 +77,17 @@ module RunoffMod
      real(r8), pointer :: inundwf(:)       ! Inundation floodplain water volume (m3)
      real(r8), pointer :: inundhf(:)       ! Inundation floodplain water depth (m)
      real(r8), pointer :: inundff(:)       ! Inundation floodplain water area fraction (no unit)
+     real(r8), pointer :: nutrDIN(:)       ! Nutrient flux for DIN
+     real(r8), pointer :: nutrDIP(:)       ! Nutrient flux for DIP
+     real(r8), pointer :: nutrDON(:)       ! Nutrient flux for DON
+     real(r8), pointer :: nutrDOP(:)       ! Nutrient flux for DOP
+     real(r8), pointer :: nutrDOC(:)       ! Nutrient flux for DOC
+     real(r8), pointer :: nutrPP(:)        ! Nutrient flux for PP
+     real(r8), pointer :: nutrDSi(:)       ! Nutrient flux for DSi
+     real(r8), pointer :: nutrPOC(:)       ! Nutrient flux for POC
+     real(r8), pointer :: nutrPN(:)        ! Nutrient flux for PN
+     real(r8), pointer :: nutrDIC(:)       ! Nutrient flux for DIC
+     real(r8), pointer :: nutrFe(:)        ! Nutrient flux for Fe
 
```

Comment:
The units of these variables should be added.

Reply:
@liho745 , i'll add units.